lib/models/models.dart

```diff
@@ -6,8 +6,9 @@ class ProductModel extends HiveObject {
   final String name;
   final String picture;
   final double price;
+  final String id;
 
-  ProductModel({this.name, this.picture, this.price});
+  ProductModel({this.name, this.picture, this.price, this.id});
```

Comment:
I think we should store "Currency" as well, with SEEDS an option, and all our Fiat Currencies an option - e.g products could be priced in Fiat, and will then be converted to Seeds based on day rate when they are being used.